### PR TITLE
Afsaneh/fix mt5

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**
ailed workflow: 
we need to update change MT5 password test based on the changes of chevron
failed workflow: https://github.com/regentmarkets/e2e-deriv-app/actions/runs/9024665107
clickup card: https://app.clickup.com/t/20696747/WALL-4053
worklfow: https://github.com/regentmarkets/e2e-deriv-app/actions/runs/9025464396

### CODE REVIEW

File: cypress/support/commands/common.js [hunk 1]
```diff
@@ -369,12 +369,12 @@ Cypress.Commands.add(
...
-              .last()
-              .should('be.visible')
-              .parent()
-              .children()
-              .contains('a', Cypress.config('baseUrl'))
+            cy.contains('p', `${accountEmail}`).last().should('be.visible')
+            // .parent()
+            // .children()
+            // .contains('a', Cypress.config('baseUrl'))
+            // .invoke('attr', 'href')
```
Comment (lubega-deriv) [inline cypress/support/commands/common.js:376]: Remove commented out code

File: cypress/e2e/wip/Wallets/mt5ResetPassword.cy.js [hunk 2]
```diff
@@ -60,10 +61,11 @@ describe('QATEST-99774 - MT5 reset password', () => {
     cy.findByText('CFDs', { exact: true }).should('be.visible')
     changeMT5Password()
   })
-  it('should be able to change mt5 password in responsive', () => {
+  it.only('should be able to change mt5 password in responsive', () => {
```
Comment (lubega-deriv) [inline cypress/e2e/wip/Wallets/mt5ResetPassword.cy.js:64]: Why is only added here?

File: cypress/e2e/wip/Wallets/mt5ResetPassword.cy.js [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-    .then(($text) => {
-      cy.log($text)
-      const buttonText = $text.trim()
-      if (buttonText === 'Open') {
-        cy.findAllByRole('button', { name: 'Open' }).eq(0).click()
+  cy.findByText('This account offers CFDs on derived instruments.')
+    .should(() => {})
+    .then((el) => {
+      if (el.length) {
+        cy.log('no MT5 account exisy')
```
Comment (lubega-deriv) [inline cypress/e2e/wip/Wallets/mt5ResetPassword.cy.js:8]: Small nitpick:
```suggestion
        cy.log('no MT5 account exist')
```

**Review by lubega-deriv — APPROVED** [review]

File: cypress/e2e/wip/Wallets/addcryptoWallet.cy.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ function addcryptowallet() {
     })
 }
 
-describe('QATEST-98773 - Add crypto wallet account', () => {
+describe('QATEST-98773 - Add crypto wallet account and QATEST-124514 wallet account switcher', () => {
```
Comment (meeyun-deriv) [inline cypress/e2e/wip/Wallets/addcryptoWallet.cy.js:42]: why we add QATEST-124514 wallet account switcher but the test does not involve wallet account switcher?

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**